common/src/main/webapp/less/style.less

```diff
@@ -1370,7 +1371,12 @@ html {
 }
 
 .ui.button.delete-button.obs-delete-button {
-  margin-left: 0;
+  margin: 0;
+  padding: 0 !important;
+
+  svg {
+    margin: 0 !important;
```

Comment:
Weird that you need an `important` here

Reply:
There's another style from SUI that seems to take precedence otherwise... Maybe there's another way around this?

![image](https://user-images.githubusercontent.com/1895643/123292742-bbe8d980-d4e9-11eb-92bc-71eba51896b1.png)